src/syscalls.csv

```diff
@@ -0,0 +1,468 @@
+# name, x86_64, arm, aarch64, mips64n32, mips64, mips, parisc, ppc64, ppc, s390, s390x, x32, x86, riscv64
```

Comment:
I ran several architectures through `arch-syscall-validate`, but not all.  (Also note to self, riscv64 isn't in `arch-syscall-validate` yet.)  Most looked good, but see comment above about mips64.

Reply:
thanks for noticing it!

Indeed there were some issues in the generated csv file.

I've fixed and verified it for each architecture